pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -38,7 +39,7 @@ type framework struct {
 var _ = Framework(&framework{})
 
 // NewFramework initializes plugins given the configuration and the registry.
-func NewFramework(r Registry, _ *runtime.Unknown) (Framework, error) {
+func NewFramework(r Registry, args []config.PluginConfig) (Framework, error) {
```

Comment:
We need to pass both `Plugins` and `PluginConfig` to NewFramewok and change the initialization for loop to build only those `Plugins` whose names appear in `Plugins`.

Reply:
Updated. PTAL.
Thanks!!